app/controllers/miq_policy_controller.rb

```diff
@@ -25,15 +25,17 @@ def index
   end
 
   def new
-    miq_policy_edit
     drop_breadcrumb(:name => _("Add New %{table}") % {:table => ui_lookup(:table => table_name)},:url  => "/#{controller_name}/new")
+    miq_policy_edit
   end
 
   def edit
     @_params[:id] ||= find_checked_items[0]
     miq_policy_edit
-    drop_breadcrumb(:name => _("Edit %{object_type} '%{object_name}'") % {:object_type => ui_lookup(:tables => table_name), :object_name => @policy.name},
-    :url  => "/#{controller_name}/#{@policy.id}/edit")
+    if params[:id]
+      @policy = MiqPolicy.find(params[:id])
+      drop_breadcrumb(:name => _("Edit %{object_type} '%{object_name}'") % {:object_type => ui_lookup(:tables => table_name), :object_name => @policy.description},:url  => "/#{controller_name}/#{@policy.id}/edit")
```

Comment:
Minor style, but we can do in a followup

```suggestion
      drop_breadcrumb(:name => _("Edit %{object_type} '%{object_name}'") % {:object_type => ui_lookup(:tables => table_name), :object_name => @policy.description}, :url  => "/#{controller_name}/#{@policy.id}/edit")
```

or even better

```suggestion
      drop_breadcrumb(
        :name => _("Edit %{object_type} '%{object_name}'") % {:object_type => ui_lookup(:tables => table_name), :object_name => @policy.description},
        :url  => "/#{controller_name}/#{@policy.id}/edit"
      )
```